consul-lambda-registrator/extension/extension.go

```diff
@@ -0,0 +1,312 @@
+package extension
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"net"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+
+	"github.com/hashicorp/terraform-aws-consul-lambda-registrator/consul-lambda-registrator/proxy"
+	"github.com/hashicorp/terraform-aws-consul-lambda-registrator/consul-lambda-registrator/structs"
+	"github.com/hashicorp/terraform-aws-consul-lambda-registrator/consul-lambda-registrator/trace"
+)
+
+type Config struct {
+	MeshGatewayURI      string        `envconfig:"MESH_GATEWAY_URI" required:"true"`
+	ExtensionDataPrefix string        `envconfig:"EXTENSION_DATA_PREFIX" required:"true"`
+	Datacenter          string        `envconfig:"DATACENTER"`
```

Comment:
What is datacenter used for? I noticed that it wasn't in the RFC.

Reply:
Datacenter is not used for the extension, it should be removed. It's only required for the Lambda registrator.

Reply:
I've removed the datacenter config from the extension.